tetrateci/iop-gke-integration.yml

```diff
@@ -0,0 +1,75 @@
+apiVersion: install.istio.io/v1alpha1
```

Comment:
I think this is used for running cni in GKE. We can't have a static file. This file could change in upstream. So we need to dynamically copy and add the values section for cni.

Reply:
If the file changes upstream, there are chances that changing it might fail. We can patch that with a diff or append the values with bash, for any better solution we would need to go to python, I think.

Reply:
File an issue. We need this addressed. We can edit the file on the fly to add the value flag. It can be addressed in another PR

Reply:
https://github.com/tetratelabs/getistio/issues/76